compose-imageviewer/gradle.properties

```diff
@@ -21,7 +21,7 @@ org.jetbrains.compose.experimental.wasm.enabled=true
 # region Versions for Kotlin 1.9.x Generation
 kotlin.version.1.9.x=1.9.21
 compose.compiler.version.1.9.x=1.5.4
-compose.wasm.version.1.9.x=1.6.0-alpha01
+compose.wasm.version.1.9.x=1.6.0
```

Comment:
jetsnack project misses such a change

Reply:
Let's add  this here as well:
```
kotlin.version.2.0.0=2.0.0-Beta4
compose.compiler.version.2.0.0=1.5.9-kt-2.0.0-Beta4
compose.wasm.version.2.0.0=1.6.0
```
___

I see it doesn't work with Beta4, unfortunately:
```
[webpack-cli] Failed to load '.../kotlin-wasm-examples/compose-imageviewer/build/js/packages/imageviewer/webpack.config.js' config
[webpack-cli] Error: ENOENT: no such file or directory, open '.../compose-imageviewer/build/js/packages/imageviewer/kotlin/imageviewer.map'
[webpack-cli] Error: ENOENT: no such file or directory, open '.../compose-imageviewer/build/js/packages/imageviewer/kotlin/imageviewer.map'
    at Object.openSync (node:fs:581:18)
    at Object.readFileSync (node:fs:457:35)
```

But It might be an issue in Beta4 (since it works with 1.9.22)